app/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,7 @@
     <string name="title_activity_event_picking">EventPickingActivity</string>
     <string name="concert_no_artist">Unknown artist</string>
     <string name="concert_no_genre">Unknown genre</string>
-    <string name="concert_no_description">Sorry ! There is no description for this concert :/</string>
+    <string name="concert_no_description">Sorry ! There is no description for this scheduledItem :/</string>
```

Comment:
Maybe no need to refactor concert to ScheduleItem in the string :)

Reply:
Yep indeed.